.travis.yml

```diff
@@ -19,7 +19,6 @@ before_script:
   - composer self-update
   - composer install
   - ./bin/jackrabbit.sh
-  - php bin/jackrabbit.php
```

Comment:
This file simply created the workspace (manually instantiating the PHPCR connection). This is now handled by the initializer.